### PR TITLE
Use v3io-py for v3io calls.

### PR DESCRIPTION
**Pull request by gtopper**

### CODE REVIEW

File: storey/flow.py [hunk 1]
```diff
@@ -1015,50 +973,63 @@ def _get_attributes_as_blob(self, aggregation_element):
...
+        if response.status_code != 200:
+            raise V3ioError(f'Failed to get number of shards. Got {response.status_code} response: {response.body}')
+        return response.output
+
+    async def _put_records(self, container, stream_path, payload):
+        self._lazy_init()
+
+        return await self._v3io_client.stream.put_records(container, stream_path, payload)
+
+    async def _get_item(self, container, stream_path, key, attributes):
```
Comment (talIguaz) [inline storey/flow.py]: `stream_path` -> `table_path`

File: storey/flow.py [hunk 2]
```diff
@@ -1015,50 +973,63 @@ def _get_attributes_as_blob(self, aggregation_element):
...
+            parsed_response = response.output.item
+            for name in parsed_response.items():
                 if name.startswith(self._aggregation_attribute_prefix):
                     del parsed_response[name]
             return parsed_response
         else:
-            raise V3ioError(f'Failed to get item. Response status code was {response.status}: {body}')
+            raise V3ioError(f'Failed to get item. Response status code was {response.status_code}: {response.body}')
+
+    async def _describe(self, container, stream_path):
```
Comment (talIguaz) [inline storey/flow.py:969]: Why half of the functions accept `container` and `path` and some a combined `path`?

Comment (gtopper) [reply storey/flow.py:969]: Because once again we're forcecd to make this separation 😿

File: storey/flow.py [hunk 3]
```diff
@@ -941,27 +942,44 @@ async def _load_schema(self, table_path):
...
+    def _build_update_expression(self, aggregation_element, additional_data):
+        expressions = []
         for name, bucket in aggregation_element.aggregation_buckets.items():
             # Only save raw aggregates, not virtual
             if bucket.should_persist:
                 blob = pickle.dumps(bucket.to_dict())
-                data[f'{self._aggregation_attribute_prefix}{name}'] = blob
-
-        return data
+                base64_blob = base64.b64encode(blob).decode('ascii')
```
Comment (talIguaz) [inline storey/flow.py:981]: Shouldn't that be without the base64encode?

File: storey/flow.py (hunk 2 above)
Comment (talIguaz) [reply storey/flow.py:969]: Why is that? it's redundant to split the path in the "calling" flows if it's only used here anyways?
Or alternatively if you want you can split that path always outside the driver.. no?

**Review by talIguaz — APPROVED** [review]

File: integration/integration_test_utils.py [hunk 4]
```diff
@@ -9,12 +10,76 @@
 import string
 import pytest
 
-from storey import NeedsV3ioAccess
 from storey.flow import V3ioError
 
 _non_int_char_pattern = re.compile(r"[^-0-9]")
 
 
+class V3ioHeaders:
```
Comment (talIguaz) [inline integration/integration_test_utils.py]: Shouldn't it inherit from `NeedV3ioAccess` to not duplicate code

**Pull request merged**